scripts/zones/Batallia_Downs/IDs.lua

```diff
@@ -23,41 +23,41 @@ zones[xi.zone.BATALLIA_DOWNS] =
         LOGIN_NUMBER                  = 7025,  -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
         MEMBERS_LEVELS_ARE_RESTRICTED = 7045,  -- Your party is unable to participate because certain members' levels are restricted.
         UNABLE_TO_PROGRESS            = 7069,  -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
-        CONQUEST_BASE                 = 7082,  -- Tallying conquest results...
-        FISHING_MESSAGE_OFFSET        = 7241,  -- You can't fish here.
-        DIG_THROW_AWAY                = 7254,  -- You dig up <item>, but your inventory is full. You regretfully throw the <item> away.
-        FIND_NOTHING                  = 7256,  -- You dig and you dig, but find nothing.
-        SPARKLING_LIGHT               = 7350,  -- The ground is sparkling with a strange light.
-        EVIL_PRESENSE                 = 7438,  -- You sense an evil Presense
-        SENSE_SOMETHING_LURKING       = 7440,  -- You sense something lurking close by!
-        NO_GRASS_GROWING_HERE         = 7488,  -- There is no grass growing here...
-        BEING_ATTACKED                = 7625,  -- You are being attacked!
```

Comment:
`EVIL_PRESENSE` was removed but we use this for `DRG_AF2_Chasing_Quotas.lua`
`BEING_ATTACKED` name was changed -> the name needs to be updated inside `A_Chocobos_Tale.lua`

The new ID should be verified with POLUtils or the update extractor.